app/models/service_template_transformation_plan_task.rb

```diff
@@ -233,6 +233,16 @@ def get_conversion_state
     update_options(updates)
   end
 
+  # Applies the limits for the task.
+  # CPU and network limits are set at the host level but other limits may not.
+  def apply_virtv2v_limits
```

Comment:
I am thinking where this and forthcoming throttling code should reside.
May be a `mixin` or a sub-module of `infra_conversion_job`?

Reply:
For me, the logic should reside in the task as it has all the knowledge of the execution context. The infra_conversion_job is mainly a state machine, and should remain pretty _stupid_ and delegate any _intelligence_ to the task. As whether it should be in a mixin, I'm wondering how much code can be reused by other classes.